node/src/main/scala/coop/rchain/node/web/WebApiRoutes.scala

```diff
@@ -139,7 +141,10 @@ object WebApiRoutes {
       // Get data
 
       case req @ POST -> Root / "data-at-name" =>
-        req.handle[DataRequest, DataResponse](webApi.listenForDataAtName)
+        req.handle[DataAtNameRequest, DataAtNameResponse](webApi.listenForDataAtName)
+
+      case req @ POST -> Root / "data-at-par" =>
```

Comment:
I don't like the name `data-at-par`. It's wrong because we accept only unforgeable name (RhoUnforg) and Par has no direct connection to WebAPI types.

Some suggestion I have in mind:
- data-at-name-block
- data-at-name-v2
- data-at-name-direct
- get-name-data
- get-channel-data
- data-at-channel

@zsluedem, @nzpr

_We will unify API interface and types for gRPC and HTTP so I don't consider this name as final for API v2._

Reply:
Let's use the same pattern as exploratory deploy and call the method `data-at-name-by-block-hash`. And the parameters are the same except term/name field.

Reply:
Done in [commit](https://github.com/rchain/rchain/pull/3584/commits/a243fb16cf1305804a6530f57653831fc4fca122). Is parameter `usePreStateHash` needed? It doesn't used in `getDataAtPar`.